aip/0122.md

```diff
@@ -196,10 +197,10 @@ example: `display_name`).
 
 ### Fields representing a resource's parent
 
-When defining a method that retrieves resources from a collection or adds a
-new resource to a collection (such as `ListBooks` or `CreateBook`), the first
-field of the request message **should** be of type `string` and **should** be
-called `parent` for the resource name of the collection.
+When defining a method that retrieves resources from a collection or adds a new
+resource to a collection (such as `ListBooks` or `CreateBook`), the first field
+of the request message **should** be of type `string` and **should** be called
+`parent` for the resource name of the collection.
 
 ```proto
 // Request message for ListBooks.
```

Comment:
Line 221 - I question the word "unrelated" here. An unrelated resource wouldn't be referenced at all - a topic *is* related to a subscription, for example. Possibly change "an unrelated" to "another"?

Line 222/223: "The field name should be equivalent to the corresponding message's name in snake case" - I'm not sure about this. It feels too much like we're suggesting Hungarian notation. The noun may not say enough about the relationship. Often it's fine, but the *role* that resource plays would generally be better, IMO. We should discuss.